Init/#1-프로젝트 초기 세팅

edhcoding opened the pull request:
## 🏷️ 이슈 번호

- close #1 

## 🧱 작업 사항

초기 기술 스택
- Next.js 15 app router
- TailwindCSS, PostCSS
- React Query v5
- Zustand

작업 사항
- next.js 및 tailwindCSS 설치
- yarn 설치
- zustand 설치 및 세팅
- tanstack query 설정
- tailwindCSS 설정
- eslint, prettier설정
- 전역 스타일링 설정
- test 환경 설정 (jest)

## 📸 결과물

## 💬 공유 포인트 및 논의 사항

i-nooo-2 committed:
Init : 프로젝트 초기 세팅

- Next.js 15 app router
- TailwindCSS, PostCSS
- Zustand
- Tanstack Query
설치 및 세팅

i-nooo-2 committed:
Init : axios 및 tanstack query, jest 세팅

- axios 설치 및 instance 생성
- server cookie func 선언
- tanstack query provider 및 devtools 설정
- jest test 환경 설정

edhcoding committed:
Init: 프로젝트 초기 세팅 옵션 수정

eunbinnie commented on app/layout.tsx:
```diff
@@ -0,0 +1,17 @@
+  children: React.ReactNode;
+}>) {
+  return (
+    <html lang="en">
```
ko로 바꾸면 좋을 것 같습니다!

eunbinnie commented on public/next.svg:
현재 안쓰는 next.svg, vercel.svg, favicon.ico는 삭제하면 좋을 것 같습니다!

eunbinnie reviewed (approved):
초기세팅 고생 많으셨습니다!!! 달려보자구욧 👍🏻 초기세팅이니 우선 승인하겠습니다!

i-nooo-2 committed:
Fix : 예시 assets 삭제 및 HTML lang attribute 변경, eslint 설정 변경

- 기본으로 제공되었던 assets 파일 제거 (favicon.ico, next.svg, vercel.svg)
- HTML lang attribute "en"에서 "ko"로 업데이트
- eslint) as'를 사용한 타입 단언 경

hongseoha reviewed (approved):
수고하셨습니다! 저도 초기세팅이니 승인 해 두겠습니다. public에 이미지나 아이콘은 작업자가 각자 추가하면 될까요?

yubugi commented:
고생하셨습니다! 저도 확인했습니다!

i-nooo-2 committed:
Chore : 색 수정 및 tailwindCSS 커스텀 추가 설정등

- 피그마 내 'nomad-black' 색이 잘못 쓰여있는 것을 확인하고, 정확한 색(#112211)으로 변경, 적용함.
- 피그마 내 여러 폰트가 사용되는 것을 확인함. 대응할 수 있도록 폰트패밀리에 스포카 한스 산스, Inter 추가
- breakpoint 추가
- 빈 디렉토리일 시 커밋에 안 올라가는 경우 확인. 더미 파일을 넣어 함께 올림.
